plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/pom.xml

```diff
@@ -11,56 +11,4 @@
   <artifactId>com.google.cloud.tools.eclipse.appengine.standard.java8</artifactId>
   <version>0.1.0-SNAPSHOT</version>
   <packaging>eclipse-plugin</packaging>
-
-  <dependencies>
```

Comment:
This prevents the in-bundle unit tests from running.  Wouldn't this be better as part of #1847?